### ISSUE TITLE
java.lang.NumberFormatException: For input string: "1-1"

### ISSUE THREAD

**Issue by 7hatGuyAJ** [X-Niter/BuycraftX#4]
Jar: tebexforged-1.20.1-1.0.0
Forge Version: 47.2.21 
Minecraft Version: 1.20.1

Encountered an unexpected exception

java.lang.NumberFormatException: For input string: "1-1"
at java.lang.NumberFormatException.forInputString(NumberFormatException.java:67) ~[?:?] {}
at java.lang.Integer.parseInt(Integer.java:665) ~[?:?] {}
at java.lang.Integer.parseInt(Integer.java:781) ~[?:?] {}
at com.github.xniter.tebexio.shared.util.VersionUtil.isVersionGreater(VersionUtil.java:45) ~[tebexforged-1.20.1-1.0.0.jar%23113!/:1.20.1-1.0.0] {re:classloading}
at com.github.xniter.tebexio.util.VersionCheck.verify(VersionCheck.java:43) ~[tebexforged-1.20.1-1.0.0.jar%23113!/:1.20.1-1.0.0] {re:classloading}
at com.github.xniter.tebexio.TebexForged.onServerStarting(TebexForged.java:150) ~[tebexforged-1.20.1-1.0.0.jar%23113!/:1.20.1-1.0.0] {re:classloading}
at com.github.xniter.tebexio.__TebexForged_onServerStarting_ServerStartingEvent.invoke(.dynamic) ~[tebexforged-1.20.1-1.0.0.jar%23113!/:1.20.1-1.0.0] {re:classloading,pl:eventbus:B}
at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.5.jar%2352!/:?] {}
at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%2352!/:?] {}
at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%2352!/:?] {}
at net.minecraftforge.server.ServerLifecycleHooks.handleServerStarting(ServerLifecycleHooks.java:110) ~[forge-1.20.1-47.2.21-universal.jar%23120!/:?] {re:classloading}
at net.minecraft.server.dedicated.DedicatedServer.m_7038_(DedicatedServer.java:201) ~[server-1.20.1-20230612.114412-srg.jar%23115!/:?] {re:classloading,pl:accesstransformer:B}
at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:634) ~[server-1.20.1-20230612.114412-srg.jar%23115!/:?] {re:classloading,pl:accesstransformer:B}
at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[server-1.20.1-20230612.114412-srg.jar%23115!/:?] {re:classloading,pl:accesstransformer:B}
at java.lang.Thread.run(Thread.java:1623) ~[?:?] {}

**Comment by X-Niter** [comment]
Hey @7hatGuyAJ , Thanks for the report, 
I see you're using a newer version of Minecraft that I have not released a supported version for.
However, I am working on releasing a 1.20.x build today! Hopfully CurseForge accepts it in the same day as well.

**Comment by 7hatGuyAJ** [comment]
Thanks for the heads up! I'm really eager to test out the new 1.20.x build.

**Comment by X-Niter** [comment]
@7hatGuyAJ  Just waiting on CurseForge to push it and it will be ready to go!

**Comment by X-Niter** [comment]
This issue has been resolved, if anything else comes up, you can make another issue and refer to this one or just make a new issue, :)
Thank you for enjoying my work/content.
